target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -61,7 +61,8 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x5000 0x844
 		ath10k_patch_mac $(mtd_get_mac_ascii bdcfg wlanmac_a)
 		;;
-	dlink,dir-859-a1)
+	dlink,dir-859-a1|\
+	dlink,dir-869-a1)
 		caldata_extract "art" 0x5000 0x844
 		ath10k_patch_mac $(mtd_get_mac_ascii devdata "wlan5mac")
```

Comment:
Same as 10-ath9k-eeprom. Please refer to 8ccbc95d5021a638d6c6611e04dcbec15aae2379.